sys/include/net/gnrc.h

```diff
@@ -7,12 +7,13 @@
  */
 
 /**
- * @defgroup    net_ng_netbase  Network Stack Pseudo Header
+ * @defgroup    net_gnrc    Generic (gnrc) network stack.
  * @ingroup     net
+ * @brief       RIOT's modular default network stack.
```

Comment:
But currently only IPv6 is implemented and I doubt that there's ever gonna be, let's say, a CCN implementation for gnrc. So, I would put at least the word IP here.